docs/internals/contributing/writing-documentation.txt

```diff
@@ -210,7 +210,7 @@ Our policy for new features is:
 
 Our preferred way for marking new features is by prefacing the features'
 documentation with: "``.. versionadded:: X.Y``", followed by a mandatory
-blank line and an optional content (indented).
+blank line and an optional description (indented).
```

Comment:
I still think there's a grammatical issue there but I'm happy to be guided by you. Perhaps changing it to "add optional content" would serve both cases?